### ISSUE TITLE
can not launch Multipass any more on macOS Ventura 

### ISSUE THREAD

**Issue by bastos1971** [canonical/multipass#2914]
Hello

I do not know why, I can not launch Multipass anymore from my MacBook Pro M1 Ventura. I have used Multipass without any issues for months before this bug.

The icons Multipass blink and shows "failing retrinving instance" et if I try to launch from CLI 

bastien@bastiens-MBP ~ % multipass list
list failed: The client is not authenticated with the Multipass service.
Please use 'multipass authenticate' before proceeding.

bastien@bastiens-MBP ~ % multipass set local.passphrase
Please enter passphrase: 
Please re-enter passphrase: 
set failed: The client is not authenticated with the Multipass service.
Please use 'multipass authenticate' before proceeding.

It is not possible to change de paraphrase. I have uninstall/reinstall Multipass without succès (from Brew and the Multipass site with dmg)

Firewall is desactived

Any Idea ? 

Thanks in advance

Bastien

**Comment by townsend2010** [comment]
Hi @bastos1971!

I'm sorry you are having issues running Multipass.  I'm a bit confused about what happened in your particular case.

Did Multipass all of a sudden just stop working with this error, like you didn't update Multipass or macOS or was this after an update of something?

Does using `sudo multipass list` work?

In the meantime, I will work on some instructions that will hopefully fix this for you.  I will post that later.

Thanks!

**Comment by jordanbaird** [comment]
I was having this same issue on 1.10.1, although upgrading to 1.11.1 seems to have fixed it

**Comment by bastos1971** [comment]
Hello jordnbaird, 

I use the last version : 

[multipass-1.11.1+mac-Darwin.pkg](https://github.com/canonical/multipass/releases/download/v1.11.1/multipass-1.11.1+mac-Darwin.pkg)

**Comment by bastos1971** [comment]
Hello

I have uninstalled and reinstalled Multipass several time last days, that the reason why I have the last version. I had the 1.10 version when the bug appeared.   Even if I reinstall Multipass the bug is still occurring

It seams that my issue is a rest of an older version that blocks. I do not know how I can remove all trace of the old version. 

***@***.*** ~ % multipass version
multipass   1.11.1+mac
***@***.*** ~ % uname -a
Darwin xxx-MBP 22.3.0 Darwin Kernel Version 22.3.0: Thu Jan  5 20:48:54 PST 2023; root:xnu-8792.81.2~2/RELEASE_ARM64_T6000 arm64

When I open the primary shell just after an installation from scratch, I have this issue. 

shell failed: The client is not authenticated with the Multipass service.
Please use 'multipass authenticate' before proceeding.

Saving session...
...copying shared history...
...saving history...truncating history files...
...completed.

[Opération terminée]

I have tried your commands, but without success… :-(

I just made a test. I have update with the dmg find on your git, my second MacBook Pro Intel from 1.10 to 3.11 with success. 

I have noticed a différence between my M1 and Intel Macbook installation. During the installation process, with my M1 MAC, the installation tools pass directly from license to type of installation but it does not ask me the location where I want to install it. Does it make any change 

My feeling is that the Multipass is not uninstalled correctly and a corrupt file/mount prevent a good installation from scratch and the 10.1 but is still on my Mac. 


Thanks for you help

Bastien

**Comment by townsend2010** [comment]
Hi @bastos1971,

This is very strange.  Could you please try what I proposed at https://github.com/canonical/multipass/issues/2549#issuecomment-1123760114?

If that doesn't work, could you please answer the following questions.

1. Did you set `MULTIPASS_STORAGE` as described [here](https://multipass.run/docs/configure-multipass-storage#heading--macos)?
2. Is it possible you have another user logged in to your Mac with a running Multipass process?  Could you please paste the output of the following command:
`$ ps -aef | grep ultipass`
(yes, no `m` is intentional)

Thank you!

**Comment by townsend2010** [comment]
Hi @ddfznt!

I'm glad that solution worked for you.  I'd like to ask if you saw this behavior when upgrading Multipass from 1.10.1 to 1.11.0 or what may have happened to trigger this behavior.

**Comment by ddfznt** [comment]
@townsend2010 can't confirm that it was the version update. I had issues with mount points.
After trying the solution here (https://github.com/canonical/multipass/issues/2914#issuecomment-1422920599) I got the error: shell failed: The client is not authenticated with the Multipass service.
After running this particular command `sudo cp ~/Library/Application\ Support/multipass-client-certificate/multipass_cert.pem /var/root/Library/Application\ Support/multipassd/authenticated-certs/multipass_client_certs.pem` I was able to create new ones vms.

But I lost access to the previously created vms.
I've consulted the log and it looks like the problem is related to qemu.

I'm using version:
➜ multipass version
multipass 1.12.0-dev.189.pr502+gd9314d80.mac
multipassd 1.12.0-dev.189.pr502+gd9314d80.mac

test is the newest instance (after install 1.12.0-dev...)
redmine-app is the old instance (created on 1.10.1)

```[2023-02-09T09:00:21.352] [info] [VMImageHost] Did not find any supported products in "appliance"
[2023-02-09T09:00:21.355] [debug] [qemu-system-aarch64] [28650] started: qemu-system-aarch64 --version
[2023-02-09T09:00:21.368] [debug] [qemu-img] [28651] started: qemu-img info /var/root/Library/Caches/multipassd/qemu/vault/images/jammy-20230107/ubuntu-22.04-server-cloudimg-arm64.img
[2023-02-09T09:00:21.374] [debug] [qemu-img] [28652] started: qemu-img resize /var/root/Library/Application Support/multipassd/qemu/vault/instances/test/ubuntu-22.04-server-cloudimg-arm64.img 21474836480
[2023-02-09T09:00:21.404] [debug] [qemu-img] [28653] started: qemu-img snapshot -l /var/root/Library/Application Support/multipassd/qemu/vault/instances/test/ubuntu-22.04-server-cloudimg-arm64.img
[2023-02-09T09:00:21.408] [debug] [test] process working dir ''
[2023-02-09T09:00:21.408] [info] [test] process program 'qemu-system-aarch64'
[2023-02-09T09:00:21.408] [info] [test] process arguments '-machine, virt,highmem=off, -accel, hvf, -drive, file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on, -cpu, cortex-a72, -nic, vmnet-shared,model=virtio-net-pci,mac=52:54:00:eb:2c:ff, -device, virtio-scsi-pci,id=scsi0, -drive, file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/test/ubuntu-22.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda, -device, scsi-hd,drive=hda,bus=scsi0.0, -smp, 1, -m, 4096M, -qmp, stdio, -chardev, null,id=char0, -serial, chardev:char0, -nographic, -cdrom, /var/root/Library/Application Support/multipassd/qemu/vault/instances/test/cloud-init-config.iso'
[2023-02-09T09:00:21.409] [debug] [qemu-system-aarch64] [28654] started: qemu-system-aarch64 -machine virt,highmem=off -nographic -dump-vmstate /private/var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/multipassd.AYfIYG
[2023-02-09T09:00:21.439] [info] [test] process state changed to Starting
[2023-02-09T09:00:21.441] [info] [test] process state changed to Running
[2023-02-09T09:00:21.441] [debug] [qemu-system-aarch64] [28655] started: qemu-system-aarch64 -machine virt,highmem=off -accel hvf -drive file=/Library/Application Support/com.canonical.multipass/bin/../Resources/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on -cpu cortex-a72 -nic vmnet-shared,model=virtio-net-pci,mac=52:54:00:eb:2c:ff -device virtio-scsi-pci,id=scsi0 -drive file=/var/root/Library/Application Support/multipassd/qemu/vault/instances/test/ubuntu-22.04-server-cloudimg-arm64.img,if=none,format=qcow2,discard=unmap,id=hda -device scsi-hd,drive=hda,bus=scsi0.0 -smp 1 -m 4096M -qmp stdio -chardev null,id=char0 -serial chardev:char0 -nographic -cdrom /var/root/Library/Application Support/multipassd/qemu/vault/instances/test/cloud-init-config.iso
[2023-02-09T09:00:21.441] [info] [test] process started
[2023-02-09T09:00:21.441] [debug] [test] Waiting for SSH to be up
[2023-02-09T09:00:21.442] [info] [VMImageHost] Did not find any supported products in "appliance"
[2023-02-09T09:00:21.470] [debug] [test] QMP: {"QMP": {"version": {"qemu": {"micro": 0, "minor": 1, "major": 7}, "package": ""}, "capabilities": ["oob"]}}

[2023-02-09T09:00:21.486] [debug] [test] QMP: {"return": {}}

[2023-02-09T09:00:32.777] [debug] [test] QMP: {"timestamp": {"seconds": 1675944032, "microseconds": 777857}, "event": "NIC_RX_FILTER_CHANGED", "data": {"path": "/machine/unattached/device[6]/virtio-backend"}}

[2023-02-09T09:00:39.317] [debug] [ssh session] Executing '[ -e /var/lib/cloud/instance/boot-finished ]'
[2023-02-09T09:00:41.002] [debug] [ssh session] Executing '[ -e /var/lib/cloud/instance/boot-finished ]'
[2023-02-09T09:00:42.194] [debug] [ssh session] Executing '[ -e /var/lib/cloud/instance/boot-finished ]'
[2023-02-09T09:00:42.239] [debug] [daemon] Returning setting local.privileged-mounts=true
[2023-02-09T09:00:55.882] [debug] [ssh session] Executing 'ip -brief -family inet address show scope global'
[2023-02-09T09:00:55.998] [debug] [ssh process] /Users/cibot/actions-runner/_work/multipass-private/multipass-private/src/ssh/ssh_process.cpp:118 read_stream(type = 0, timeout = -1): 
[2023-02-09T09:00:55.998] [debug] [ssh process] /Users/cibot/actions-runner/_work/multipass-private/multipass-private/src/ssh/ssh_process.cpp:136 read_stream(): num_bytes = 61
[2023-02-09T09:00:55.998] [debug] [ssh process] /Users/cibot/actions-runner/_work/multipass-private/multipass-private/src/ssh/ssh_process.cpp:136 read_stream(): num_bytes = 0
[2023-02-09T09:01:00.102] [debug] [ssh session] Executing 'ip -brief -family inet address show scope global'
[2023-02-09T09:01:00.145] [debug] [ssh process] /Users/cibot/actions-runner/_work/multipass-private/multipass-private/src/ssh/ssh_process.cpp:118 read_stream(type = 0, timeout = -1): 
[2023-02-09T09:01:00.145] [debug] [ssh process] /Users/cibot/actions-runner/_work/multipass-private/multipass-private/src/ssh/ssh_process.cpp:136 read_stream(): num_bytes = 61
[2023-02-09T09:01:00.145] [debug] [ssh process] /Users/cibot/actions-runner/_work/multipass-private/multipass-private/src/ssh/ssh_process.cpp:136 read_stream(): num_bytes = 0
[2023-02-09T09:01:01.255] [debug] [ssh session] Executing 'ip -brief -family inet address show scope global'
[2023-02-09T09:01:01.297] [debug] [ssh process] /Users/cibot/actions-runner/_work/multipass-private/multipass-private/src/ssh/ssh_process.cpp:118 read_stream(type = 0, timeout = -1): 
[2023-02-09T09:01:01.297] [debug] [ssh process] /Users/cibot/actions-runner/_work/multipass-private/multipass-private/src/ssh/ssh_process.cpp:136 read_stream(): num_bytes = 61
[2023-02-09T09:01:01.297] [debug] [ssh process] /Users/cibot/actions-runner/_work/multipass-private/multipass-private/src/ssh/ssh_process.cpp:136 read_stream(): num_bytes = 0
[2023-02-09T09:01:03.713] [debug] [ssh session] Executing 'ip -brief -family inet address show scope global'
[2023-02-09T09:01:03.757] [debug] [ssh process] /Users/cibot/actions-runner/_work/multipass-private/multipass-private/src/ssh/ssh_process.cpp:118 read_stream(type = 0, timeout = -1): 
[2023-02-09T09:01:03.757] [debug] [ssh process] /Users/cibot/actions-runner/_work/multipass-private/multipass-private/src/ssh/ssh_process.cpp:136 read_stream(): num_bytes = 61
[2023-02-09T09:01:03.757] [debug] [ssh process] /Users/cibot/actions-runner/_work/multipass-private/multipass-private/src/ssh/ssh_process.cpp:136 read_stream(): num_bytes = 0
[2023-02-09T09:01:06.544] [debug] [ssh session] Executing 'ip -brief -family inet address show scope global'
[2023-02-09T09:01:06.589] [debug] [ssh process] /Users/cibot/actions-runner/_work/multipass-private/multipass-private/src/ssh/ssh_process.cpp:118 read_stream(type = 0, timeout = -1): 
[2023-02-09T09:01:06.589] [debug] [ssh process] /Users/cibot/actions-runner/_work/multipass-private/multipass-private/src/ssh/ssh_process.cpp:136 read_stream(): num_bytes = 61
[2023-02-09T09:01:06.589] [debug] [ssh process] /Users/cibot/actions-runner/_work/multipass-private/multipass-private/src/ssh/ssh_process.cpp:136 read_stream(): num_bytes = 0
[2023-02-09T09:01:09.020] [debug] [test] QMP: {"return": {}}
{"timestamp": {"seconds": 1675944069, "microseconds": 20202}, "event": "POWERDOWN"}

[2023-02-09T09:01:09.687] [debug] [test] QMP: {"timestamp": {"seconds": 1675944069, "microseconds": 687376}, "event": "RTC_CHANGE", "data": {"offset": 1, "qom-path": "/machine/unattached/device[4]"}}

[2023-02-09T09:01:09.726] [debug] [test] QMP: {"timestamp": {"seconds": 1675944069, "microseconds": 726866}, "event": "SHUTDOWN", "data": {"guest": true, "reason": "guest-shutdown"}}

[2023-02-09T09:01:09.727] [info] [test] VM shut down
[2023-02-09T09:01:09.742] [info] [test] process state changed to NotRunning
[2023-02-09T09:01:09.742] [info] [test] process finished with exit code 0
[2023-02-09T10:40:40.137] [info] [daemon] Cannot open ssh session on "redmine-app" shutdown: ssh connection failed: 'Timeout connecting to 192.168.64.25'
[2023-02-09T10:40:40.140] [debug] [redmine-app] QMP: {"return": {}}
{"timestamp": {"seconds": 1675950040, "microseconds": 139935}, "event": "POWERDOWN"}

[2023-02-09T10:41:20.768] [warning] [daemon] Instance 'redmine-app' is already running, but in an unknown state
[2023-02-09T10:41:20.768] [debug] [redmine-app] Waiting for SSH to be up```

**Comment by bastos1971** [comment]
Hello

Thanks Emerson. I have applied your advice and it works now .. after 2 weeks of investigation… and good news, I could get access to all my previously created  VM’s

What I did

- Stop multipass

Run Emerson’s command 
> cp ~/Library/Application\ Support/multipass-client-certificate/multipass_cert.pem /var/root/Library/Application\ Support/multipassd/authenticated-certs/multipass_client_certs.pem
> 


- Reinstall multipass 11.1 with  pkg version download from official repo, without uninstalling the previous version 

- Miracle .. I could see all my previous VM.. 

Many thanks Emerson and Chris for your help, this prevent me to reinstall my whole Mac .-) 

Best regards
Bastien

**Comment by townsend2010** [comment]
Hello @bastos1971!

I'm so glad you got it working!  Thank you for letting us know.  I'm closing this bug report, but please open a new one if you run into any more troubles.

**Comment by bastos1971** [comment]
Hello ChrisYou can just add these topic  to complete the bug/incident. I noticed that I could see all my old VM (multipass list) after I applied the workaround. But I could not start them, because they went  into never end starting loops.  But I was able to create new VM normally.. So I have deleted/purged my old VM and create new VM instead. Now it is working well Thanks for the support Best regards Envoyé de mon iPhoneLe 11 févr. 2023 à 03:11, Chris Townsend ***@***.***> a écrit :﻿
Hello @bastos1971!
I'm so glad you got it working!  Thank you for letting us know.  I'm closing this bug report, but please open a new one if you run into any more troubles.

**Comment by jlj77** [comment]
> Hi @ddfznt!
> 
> I'm glad that solution worked for you. I'd like to ask if you saw this behavior when upgrading Multipass from 1.10.1 to 1.11.0 or what may have happened to trigger this behavior.

This also fixed a fresh (homebrew) install of 1.14.1+mac, for me, fyi.
